docs/docs/how-tos/jupyter-gallery.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Share Git Repository via JupyterLab-Gallery
+description: Share files or tutorials with your team
+---
+
+# How to set up a Git Repository via JupyterLab-Gallery
+
+[JupyterLab-Gallery ](https://github.com/nebari-dev/jupyterlab-gallery) is a JupyterLab plugin
+that allows users to share a Git Repository. JupyterLab users are presented with a tile in
+the JupyterLab launch screen. Users can choose to download the Gallery.
+
+By default, if no JupyterLab-Gallery items are added to the config, users will not see this section
+in the JupyterLab Launcher.
+
+This feature was added to Nebari in the 2024.6.1 release.
+
+## Configure JupyterLb-Gallery
+
+In Nebari, the JupyterLab-Gallery configuration is set in the Nebari configuration, `nebari-config.yaml`.
+
+The configuration is set under the `jupyterlab` section under the subsection `gallery_settings`.
+Users may set a title for the Gallery section of the Launcher screen and a destination (i.e. folder)
+for the location of the cloned repos.
+
+Several repositories can be configured to display as individual gallery tiles. Under the `exhibits`
+section, each repository must have a `title` and a `git` location. There are also several other
+optional config options.
+
+The configuration is set at two different levels. Top level gallery settings define the title of
+the Gallery section and the location in which the repositories are downloaded. The repository
+level settings are defined for each repository. The descriptions of each section are described
+below.
+
+**Top Level Gallery Settings**
+
+| Section     | Description                                        |
+| ----------- | -------------------------------------------------- |
+| title       | Title on the gallery tile                          |
+| destination | Location on disk where the repo will be downloaded |
+
+**Repository Level Settings**
+
+| Section     | Description                                                                  |
+| ----------- | ---------------------------------------------------------------------------- |
+| title       | Title on the gallery tile                                                    |
+| git         | URL of the git repository                                                    |
+| homepage    | (Optional)                                                                   |
+| description | Description of the repository to appear on the gallery tile (Optional)       |
+| icon        | base64 encoded image to use an icon to appear on the gallery tile (Optional) |
+| account     | (Optional)                                                                   |
```

Comment:
Not sure what this is?

Reply:
When you log in using `token` you need to also pass `account`. Depending on whether it is GitHub or GitLab it should be a pre-defined string,  or any string.

Reply:
This is defined in https://github.com/nebari-dev/jupyterlab-gallery/blob/ef56fc616bae4ec8305c01e3341b773ca4cfd48f/jupyterlab_gallery/manager.py#L30-L82

You can access the documentation via `jupyterlab-gallery --help-all`. I will add it to the README.

Reply:
@krassowski for `account`, what does that look like in a github and gitlab url?

For example, if I go to "clone from https" on the nebari-dev repo, it gives this url: `https://github.com/nebari-dev/nebari.git` . Would the `account` be `nebari-dev` or is that my username associate with the token?

For gitlab the "clone from https" could looks like this: `https://gitlab.<domain>.net/<projectname>/<teamname>/gradient.git`. What is the `account` here?

Reply:
See the example at the bottom of https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#using-a-personal-access-token-on-the-command-line